packages/avatar/src/constants.js

```diff
@@ -0,0 +1,16 @@
+import { sizes } from "./sizes";
+
+export const SizeMapping = {
+  [sizes.SMALL_16]: "extraSmall",
+  [sizes.MEDIUM_24]: "small",
+  [sizes.MEDIUM_32]: "medium",
+  [sizes.LARGE_48]: "large",
+  [sizes.XLARGE_64]: "extraLarge"
+};
+
+export const StyleItems = {
```

Comment:
Curious to know the reason behind making the `stylesheet` properties as variables?